src/cmdc/datasets/official/CA/data.py

```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+import textwrap
+
+from cmdc.datasets.official.base import CountyData
+
+
+CA_COUNTY_URL = "https://data.chhs.ca.gov/"
+CA_COUNTY_URL += "dataset/6882c390-b2d7-4b9a-aefa-2068cee63e47/resource/"
+CA_COUNTY_URL += "6cd8d424-dfaa-4bdd-9410-a3d656e1176e/download/covid19data.csv"
+
+C_RENAMER = {
+    "County Name": "name",
+    "Most Recent Date": "dt",
+    "Total Count Confirmed": "cases_total",
+    "Total Count Deaths": "deaths_total",
+    "COVID-19 Positive Patients": "hospital_beds_in_use_covid_confirmed",
+    "Suspected COVID-19 Positive Patients": "hospital_beds_in_use_covid_suspected",
+    "ICU COVID-19 Positive Patients": "icu_beds_in_use_covid_confirmed",
+    "ICU COVID-19 Suspected Patients": "icu_beds_in_use_covid_suspected",
+}
+
+
+class CACountyData(CountyData):
```

Comment:
What are you getting from `CountyData`?

Just the `table_name` and `pk` attributes?

Reply:
You definitely get `table_name` and `pk`, but, in other cases, you may also get the `_insert_query` method -- It just turned out that CA needed to be special cased.